build.sbt

```diff
@@ -120,6 +120,17 @@ val circe = project
   )
   .enablePlugins(AutomateHeaderPlugin)
 
+val zioJson = project
```

Comment:
Could you rename this to `zio-json` for consistency with other modules?

Reply:
You mean the val name?

Reply:
yep

Reply:
Ok done